lib/fft.d.ts

```diff
@@ -0,0 +1,14 @@
+export = FFT;
+declare function FFT(size: any): void;
+declare class FFT {
+    constructor(size: any);
+    size: number;
+    table: any[];
+    fromComplexArray(complex: any, storage: any): any;
+    createComplexArray(): any[];
+    toComplexArray(input: any, storage: any): any;
```

Comment:
Either the README documentation is incorrect, or `storage` needs to be optional.

```
    toComplexArray(input: any, storage?: any): any;
```

Reply:
Otherwise this errors out in type script `const data = fft.toComplexArray(y);`